src/actions/feedbackActions.js

```diff
@@ -33,6 +33,12 @@ export function feedbackSuccess(content, sessionId, mentorId, jwt) {
   };
 }
 
+export function cancelFeedback() {
+  return function goCancelFeedback(dispatch) {
+    return dispatch(createFeedbackSuccess());
```

Comment:
@AnkitPatanaik Why dispatch Success() in the cancel action?

Reply:
i want the same functionality from the success, however i should probably create another action function to be more explicit/clear  with what's happening

Reply:
Okay, I figured it was that. I think down the road, there will be a divergence between success and failure (i.e. when we put API calls in)